index.js

```diff
@@ -247,7 +248,14 @@ async function getSiteVersion(page) {
 async function launch(url, selector, device) {
     'use strict';
 
-    const browser = await puppeteer.launch();
+    var chrome_args = [];
+    if (config.DISABLE_CHROME_SANDBOX){
+      chrome_args = ['--no-sandbox'];
+    }
+
+    const browser = await puppeteer.launch({
+       args: chrome_args,
+    }).catch(console.error);
```

Comment:
If this does fail we shouldn't continue by printing the error to the console, i think we rather halt the execution.

i.e.

```suggestion
    const browser = await puppeteer.launch({args});
```

Reply:
I think this was an attempt to get some information around why Chromium wasn't launching successfully as the execution fails anyway whether we catch the error or not.